corehq/apps/users/tests/retire.py

```diff
@@ -328,6 +329,8 @@ def test_all_case_forms_deleted(self):
             self.assertTrue(XFormInstance.objects.get_form(form_id, self.domain).is_deleted)
 
         self.assertTrue(CommCareCase.objects.get_case(usercase_id, self.domain).is_deleted)
+        self.assertIsNone(CommCareCase.objects.get_case_by_external_id(
+            self.domain, self.commcare_user._id, USERCASE_TYPE))
```

Comment:
👍🏻

Reply:
can we add a test about the state of this case
`self.assertTrue(CommCareCase.objects.get_case(usercase_id, self.domain).closed)`
or
`self.assertFalse(CommCareCase.objects.get_case(usercase_id, self.domain).is_deleted)`
?

I don't know if not closing the case would impact anything anyway since having an open deleted case should not matter but just good to know.

Reply:
There is assertion one line above about the case deletion, if that's what you are referring to?
`self.assertTrue(CommCareCase.objects.get_case(usercase_id, self.domain).is_deleted)`

Reply:
https://github.com/dimagi/commcare-hq/pull/32503/commits/638ca124a82c9d71d4edf5110cad3d3deb626a58